lib/NormalModuleReplacementPlugin.js

```diff
@@ -43,6 +43,7 @@ class NormalModuleReplacementPlugin {
 				});
 				nmf.hooks.afterResolve.tap("NormalModuleReplacementPlugin", result => {
 					const createData = result.createData;
```

Comment:
this was undefined, I'm not sure why

Reply:
Create a reproducible repo depicting these errors.